components/rating/stories/template.js

```diff
@@ -7,12 +7,12 @@ import { ifDefined } from "lit-html/directives/if-defined.js";
 import { Template as Icon } from "@spectrum-css/icon/stories/template.js";
 
 import "../index.css";
```

Comment:
You'll need to load the express assets in the storybook as well - check out the [button storybook template](https://github.com/adobe/spectrum-css/blob/e13e1fa04265d5c826b17bf760402e4f42f764d4/components/button/stories/template.js#L27-L28) for an example (line 27).